firestore/integration_test_internal/src/filter_test.cc

```diff
@@ -39,6 +39,8 @@ TEST_F(FilterTest, CopyConstructorReturnsEqualObject) {
 }
 
 TEST_F(FilterTest, CopyAssignementReturnsEqualObject) {
+  GTEST_SKIP() << "b/293326241: The Android implementation is failing.";
```

Comment:
I just remembered an even better solution: `SKIP_TEST_ON_ANDROID`. Please use that instead so that the test continues to run in the other target platforms.

https://github.com/firebase/firebase-cpp-sdk/blob/b9a53cf29fefe8e351e8b6b53a238a8365aa8993/testing/test_framework/src/firebase_test_framework.h#L185